nodejs/eks/cluster.ts

```diff
@@ -1905,6 +1905,12 @@ export class ClusterInternal extends pulumi.ComponentResource {
 
         super(type, name, args, opts);
 
+        if (args?.creationRoleProvider?.provider) {
+            throw new Error(
+                "The `creationRoleProvider.provider` option is not supported in non-nodejs Pulumi programs. Please use the `providerCredentialOpts` option instead.",
+            );
+        }
```

Comment:
Is there an issue for serializing resources in nodejs MLCs in Pulumi? If so, can we link it. If not, can we open the feature request and link it?